src/genie/libs/parser/iosxe/show_license.py

```diff
@@ -178,3 +178,69 @@ def cli(self, output=None):
             ''' 
 
         return ret_dict
+
+# ----------------------
+# =================
+# Schema for:
+#  * 'show license summary'
+# =================
+class ShowLicenseSummarySchema(MetaParser):
+    """Schema for show license udi"""
+    schema = {
+        'license_usage': {
+            Any(): {
+                'entitlement': str,
+                'count': str,
+                'status': str,
+            }
+        }
+    }
+
+# =================
+# Parser for:
+#  * 'show license summary'
+# =================
+class ShowLicenseSummary(ShowLicenseSummarySchema):
+    """Parser for show license summary"""
+    cli_command = 'show license summary'
+    
+    """
+    License Usage:
+      License                 Entitlement Tag               Count Status
+      -----------------------------------------------------------------------------
+      network-advantage       (C9300-48 Network Advan...)       1 IN USE
+      dna-advantage           (C9300-48 DNA Advantage)          1 IN USE
+    """
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        # initial return dictionary
+        license_summ_dict = {}
+        
+        result_dict = {}
+
+        # license entitlement count status
+        p0 = re.compile(r"^(?P<license>network-\w+|dna-\w+)\s+(?P<entitlement>\(\w+-\d+\s\w+\s\w+.+\))\s+(?P<count>\d+)\s+(?P<status>.+)$")
```

Comment:
While this regex pattern does match the output you've provided, it looks like it might struggle to generalize well to other variations of device output. 

What do you think of this regex pattern?
```
^(?P<license>[\w-]+)\s+\((?P<entitlement>.+)\)\s+(?P<count>\d+)\s+(?P<status>.+)$
```
![image](https://user-images.githubusercontent.com/33130026/128190977-76fd8b31-1474-4fd6-b705-331eab9dd6fd.png)

Reply:
I agree! I wasn't sure how specific to make the regex. Thanks!